tests/components/lovelace/test_dashboard.py

```diff
@@ -225,6 +224,25 @@
 
     client = await hass_ws_client(hass)
 
+    # List dashboards
+    await client.send_json({"id": 4, "type": "lovelace/dashboards/list"})
+    response = await client.receive_json()
+    assert response["success"]
+    assert len(response["result"]) == 2
+    with_sb, without_sb = response["result"]
+
+    assert with_sb["mode"] == "yaml"
+    assert with_sb["filename"] == "bla.yaml"
+    assert with_sb["sidebar"] == {"title": "Test Panel", "icon": "mdi:test-icon"}
+    assert with_sb["require_admin"] is True
+    assert with_sb["url_path"] == "test-panel"
+
+    assert without_sb["mode"] == "yaml"
+    assert without_sb["filename"] == "bla2.yaml"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5093948)